build/IPDK_Container/scripts/build_p4sde.sh

```diff
@@ -1,11 +1,12 @@
 #Copyright (C) 2021 Intel Corporation
 #SPDX-License-Identifier: Apache-2.0
 
-#!/usr/bin/bash
+#!/bin/bash
```

Comment:
This line should be placed as first line in the script

Reply:
fixed in the scripts I have modified.